### DISCUSSION TITLE
blog/2023/230113-my-2022-website/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#9] [Announcements]
# blog/2023/230113-my-2022-website/

在浏览器中输入一个字母，它将自动补全你访问最多的一个网站。一起来回顾一下 2022 年浏览器的 A to Z 吧。新的一年到来，就用这篇 Blog 开启 2023 的新篇章。

https://blog.arisa.moe/blog/2023/230113-my-2022-website/

**Comment by zayze-dev** [comment]
非常有趣ww，感谢分享
